### DISCUSSION TITLE
How about using GitHub issues and Projects to manage workload

### DISCUSSION THREAD

**Discussion by jasonBirchall** [ministryofjustice/analytics-platform#140] [General]
We use GitHub for most things in our day to day development, be it hosting code, or reviewing PR's. We should use the `analytics-platform` repository to record our backlog of issues and projects to manage and review.

**Comment by roblevi** [comment]
I'm no fan of Jira - but don't see the benefit of moving project management to GitHub to be big enough to do that work. Also not everyone in the team works with GitHub day to day. We produce metrics off the Jira for project governance that would have to be worked through too.
